src/TypeScript/EslintBridgeClient/Contract/InitLinterRequest.cs

```diff
@@ -38,7 +38,11 @@ internal class Rule
     {
         [JsonProperty("key")]
         public string Key { get; set; }
+
         [JsonProperty("configurations")]
         public object[] Configurations { get; set; }
+
+        [JsonProperty("fileTypeTarget")]
+        public string[] FileTypeTarget { get; set; }
```

Comment:
The parameter has been added to the rule metadata to describe which type of files it applies to.

There is a corresponding change on the "analyze file" method that we have not implemented i.e. we should be passing in the type of the file being analyzed.
See here: https://github.com/SonarSource/SonarJS/blob/master/eslint-bridge/src/analyzer.ts

The fact that analysis seems to work without us passing this parameter is probably just lucky. We don't currently have a way to decide whether a file is a test file or not, so all we could do is pass a hard-coded value. However, it would probably be safer to pass it.

Reply:
I've added another ticket to project for this.